src/target/riscv/riscv.h

```diff
@@ -12,29 +12,29 @@ struct riscv_program;
 #include "target/register.h"
 #include "target/semihosting_common.h"
 #include <helper/command.h>
+#include <helper/bits.h>
 
-#define RISCV_COMMON_MAGIC	0x52495356U
+#define RISCV_COMMON_MAGIC	(0x52495356U)
```

Comment:
Simple constants don't need parens. It's only important if your #define contains more than one token, such as in RISCV_SATP_MODE below, and the new version of RISCV_MAX_HARTS.

Reply:
done